chapters/en/chapter3/3.mdx

```diff
@@ -162,7 +162,7 @@ This time, it will report the validation loss and metrics at the end of each epo
 
 The `Trainer` will work out of the box on multiple GPUs or TPUs and provides lots of options, like mixed-precision training (use `fp16 = True` in your training arguments). We will go over everything it supports in Chapter 10.
 
-This concludes the introduction to fine-tuning using the `Trainer` API. An example of doing this for most common NLP tasks will be given in Chapter 7, but for now let's look at how to do the same thing in pure PyTorch.
+This concludes the introduction to fine-tuning using the `Trainer` API. An example of doing this for most common NLP tasks will be given in [Chapter 7](course/chapter7), but for now let's look at how to do the same thing in pure PyTorch.
```

Comment:
Nice catch!

Reply:
Yes, thank you very much. I'll make a note to correct this in my next PR.